src/cli/Cli.ts

```diff
@@ -528,17 +528,13 @@ export class Cli {
     if (options.output === 'csv') {
       const { stringify } = require('csv-stringify/sync');
 
-      /* 
-        https://csv.js.org/stringify/options/
-        header: Display the column names on the first line if the columns option is provided or discovered.
-        escape: Single character used for escaping; only apply to characters matching the quote and the escape options default to ".
-        quote: The quote characters surrounding a field, defaults to the " (double quotation marks), an empty quote value will preserve the original field, whether it contains quotation marks or not.
-        quoted: Boolean, default to false, quote all the non-empty fields even if not required.
-        quotedEmpty: Quote empty strings and overrides quoted_string on empty strings when defined; default is false.
-      */
-
+      // https://csv.js.org/stringify/options/
       return stringify(logStatement, {
-        header: true
+        header:Cli.getInstance().getSettingWithDefaultValue<boolean>(settingsNames.csvHeader, true),
```

Comment:
Instead of calling `Cli.getInstance()` multiple times, let's call it once above and use the reference later.